includes/minishell.h

```diff
@@ -13,8 +13,8 @@
 #ifndef MINISHELL_H
 # define MINISHELL_H
 
-# include <stdio.h>
```

Comment:
we need this library, its allowed library stdio.

Reply:
Ok!